### PR TITLE
Add ability to specify eventsEnabled flag

### PR DESCRIPTION
**Pull request by what-is-a-crow**

### CODE REVIEW

**Comment by mohsinulhaq** [comment]
https://github.com/popperjs/react-popper/blob/master/typings/react-popper.d.ts#L36
there are some more props that we are not accepting right now, can you add those as well? maybe even just `...restProps` the ones that are pass-through?

**Comment by what-is-a-crow** [comment]
@mohsinulhaq thanks for the quick review! Passing through the remaining props seems to do the trick as well and is simpler; good call.

File: src/TooltipTrigger.tsx [hunk 1]
```diff
@@ -82,6 +82,7 @@ class TooltipTrigger extends Component<
           }),
           ...modifiers
         }}
+        {...this.props}
```
Comment (mohsinulhaq) [inline src/TooltipTrigger.tsx]: instead, you should gather restProps from this.props destructuring above and spread that here

**Comment by what-is-a-crow** [comment]
@mohsinulhaq I didn't know about that language feature! Very cool.

**Pull request merged**